packages/grafana-data/src/utils/OptionsUIBuilders.ts

```diff
@@ -185,7 +187,22 @@ export class NestedPanelOptionsBuilder<TSub = any> implements OptionsEditorItem<
   constructor(public cfg: NestedPanelOptions<TSub>) {
     this.path = cfg.path;
     this.category = cfg.category;
-    this.defaultValue = cfg.defaultValue;
+    this.defaultValue = this.getDefaultValue(cfg) as TSub;
+  }
+
+  private getDefaultValue(cfg: NestedPanelOptions<TSub>) {
+    let result = cfg.defaultValue || {};
```

Comment:
Question: is there any risk by mutating the `cfg.defaultValue` with the `set` function at line 201? Would it be better to work against a copy of the defaultValue instead?

```suggestion
    let result = cloneDeep(cfg.defaultValue || {});
```